src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| AcquisitionVoxelSize              | RECOMMENDED                                                                                                                     | [array][] of [numbers][]                | An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| FlipAngle                         | RECOMMENDED, REQUIRED for ASL-sequences that use a Look-Locker readout and/or multiple flip angles (for example (turbo-)QUASAR) | [number][] or [array][] of [numbers][]  | Flip angle for the acquisition, specified in degrees, either as a number or an array of numbers, with the length corresponding to the number of volumes. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
This seems to conflict with other approaches to data with multiple flip angles (e.g., the `VFA` suffix proposed in BEP001), where a new `flip` entity is added and the data are separated into different files. Do `(turbo-)QUASAR` data _need_ to be in one file?

Reply:
Hi @tsalo . Indeed, the approach used for BEP005 is different from those used by BEP001. We had a discussion with the BEP001 about this before submitting the PR, and we agreed that for this case, not one size fits all. 

For their case, most volumes are already stored as separate nii files by the scanner, whereas in our case, all volumes are stored in one nii file (timeseries). Both groups agreed that the best approach is to keep the datafiles as they are acquired, so no merging, and also no splitting up. Additionally, for ASL, it is very important not to split up the file, for processing purposes (e.g. motion correction), as the data files are already very small. This applies also to QUASAR data, where, like in other multi-PLD ASL files, all data are kept in a single file as acquired.

Reply:
It does sound like this is key for ASL, but changing the format and definition of the field in the ASL section means changing it for the specification as a whole. We discussed this issue in today's BIDS maintainers call, and we don't want to change the format for a common metadata field without discussing it more in the context of other parts of the specification. 

As a hack, we're proposing that variable EchoTimes and FlipAngles _not_ be set in the json file, and instead be set in the aslcontext.tsv file as additional columns. Tagging @bids-standard/maintainers, who were involved in the proposal.

Reply:
@tsalo I will propose this in our meeting later this week.

Reply:
That's a nice suggestion. To clarify, it is not about the storage by the scanner, but it is about whether scans fall in the same or a different sequence as acquired by the scanner, and to keep acquisition order intact. I'm not sure if for QUASAR this is an absolute requirement, we chose this for consistency.